packages/components/fields/checkbox-field/docs/usage-example.js

```diff
@@ -0,0 +1,27 @@
+import CheckBoxField from '@commercetools-uikit/checkbox-field';
+
+const Example = () => (
+  <div>
+    <CheckBoxField
+      value="foo-checkbox-value"
+      onChange={(event) => alert(event.target.value)}
+      isChecked={true}
```

Comment:
Nit: might be nice to have some examples of the `FieldLabel` props.
Maybe just adding something like: 
```
    title="Welcome Text"
    hintIcon={<WarningIcon />}
    description={<FormattedMessage {...messages.description} />}
    badge={<FlatButton tone="primary" label="label" />}
```